.github/workflows/fuzz_duckdb.yml

```diff
@@ -12,7 +12,7 @@ on:
         type: number
         default: 120
       dry:
-        required: true
+        required: false
```

Comment:
To avoid if/else statements I pass here the whole parameter. For example with `--dry`, if we need it to run with that flag, we should pass the flag to the reusable workflow as it is. When a parameter has `required: false`, it becomes optional.

I see that passing parameter like `dry` as `with: dry: "--dry"` is difficult to read. And also I removed it everywhere, where it can be passed and only left in the reusable workflow as an optional parameter. Maybe I need to add a comment like `If you need to run fuzzer from your fork, please use this workflow with: dry: "--dry" parameter`?

Reply:
Passing `dry: false` is done. please also see a PR against the `duckdb/duckdb_sqlsmith` [here](https://github.com/duckdb/duckdb_sqlsmith/pull/29)

Reply:
Oops, my understanding of `dry` got mixed up. Default for `dry` should be `true`, and it should not be required. 

Eventually I will change this, as it is not positive language friendly (i.e extra parameters means extra things should happen, not the absence of other things)